apps/frontend/src/assets/css/scaffolding.css

```diff
@@ -1,3 +1,5 @@
+@import url("https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,200..1000;1,200..1000&family=Righteous&family=Ubuntu:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400;1,500;1,700&display=swap");
```

Comment:
Remove for now, we will add fonts separately

Reply:
oke, i remove now